personalsite/src/App.js

```diff
@@ -0,0 +1,54 @@
+import logo from './logo.svg';
+import './App.css';
+import React from 'react';
+
+function App() {
+
+  const name = "Emily"
+  var nameComponent = (name== "Emily") ?
+  <p>hi my name is {name}</p> : <p>my  name is not Emily</p>;
+  const buttonFunction = () => {
+    console.log("the button was clicked");
+  }
+ 
+
+  return(
+    
+    <div className='App'>
+
+    <header className="App-header">
+        <div>{nameComponent}</div>
+       <button onClick={buttonFunction}>I am a button</button>
+     </header>
+   </div>
+      </header>
+
+    <body>
+      <h1 className = "white-text">Hi, My Name is Emily! :)</h1>
+      <h2 className = "white-text">Dartmouth '24</h2>
+      <p></p>
+      <hr></hr>
+      <img src="download.jpg"/>
+      <img src="download-1.jpg"/>
```

Comment:
oh also for images in react, you want to import them in at the top using javascript eg)
`import download-1 from './download-1.jpg';`

and then you can link them to the image like `<img src={download-1}/>`

this was another thing that was crashing your code!